app/javascript/components/applications/application_form/formOptions.js

```diff
@@ -17,13 +21,13 @@ const labelize = (options, attrs = {}, noPlaceholder = false) => {
   }
 
   return labelizedOptions.concat(
-    map(options, (option) => (
-      {
+    options.map(option => {
+      return {
         value: option.hasOwnProperty('value') ? option.value : option,
-        label: option.label || option,
+        label: option.hasOwnProperty('label') ? option.label : option,
```

Comment:
This was causing a bug where null an option like this:

```
{
  value: "value",
  label: null
}
```

Would labelize into
```
{
  value: "value",
  label: {
    value: "value",
    label: null
  }
}
```

Which breaks during render